samples/meetings-content-bubble/csharp/TargetedNotifications/Pages/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Microsoft Teams Meeting content bubble sample</title>
+
```

Comment:
Should we change this name as well?
content bubble sample

Reply:
changed